build/templates/networking.tpl.json

```diff
@@ -611,11 +643,15 @@
                     },
                     {
                       "columnId": "NotSucceeded",
-                      "label": "IP Not Succeeded"
+                      "label": "Failed"
```

Comment:
Changed expression to this. The current translated Japanese on Portal is a bit strange, so I'll try changing and then check it after deployment.